### PR TITLE
Synchronize native-hello-module demo with the website version.

### PR DESCRIPTION
**Pull request by LesiaChaban**

### CODE REVIEW

File: native-hello-module/README.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
-# GraalVM Demos: Building a HelloWorld Java Module into a Native Executable
-This repository contains the code for a demo application for [GraalVM](http://graalvm.org).
+# Build Java Module into a Native Executable
+
+GraalVM Native Image supports the Java Platform Module System, introduced in Java 9, which means you can convert a modularized Java application into a native executable.
+
+The native-image tool accepts the module-related arguments like `--module (-m)`, `--module-path (-p)`, `--add-opens`, `--add-exports` (same as for the java launcher). When such a module-related argument is used, the native-image tool itself is used as a module too.
```
Comment (olyagpl) [inline native-hello-module/README.md:5]: Wrap `native-image` in code tags. Same as the `java` launcher.

File: native-hello-module/README.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+
+In addition to supporting `--add-reads` and `--add-modules`, all module related options are considered prior to scanning the modulepath. This helps prevent class loading errors and allow for better module introspection at run time.
+
+The command to build a native executable from a Java module is:
+```bush
+native-image [options] --module <module>[/<mainclass>] [options]
+```
+The example below shows how to build a modular Java application into a native executable. 
 
 ## Prerequisites
```
Comment (olyagpl) [inline native-hello-module/README.md:15]: @LesiaChaban, please remove the Prerequisites part completely. It is self-explanatory that a user must have GraalVM installed if he is in the GraalVM Demos repository. You also tell how to install GraalVM with NI in the `#Preparation` section.

Comment (olyagpl) [inline native-hello-module/README.md:15]: Please remove ## Prerequisites from other demos you have added lately.

File: native-hello-module/README.md [hunk 3]
```diff
@@ -8,47 +19,20 @@ This repository contains the code for a demo application for [GraalVM](http://gr
...
-## Build the Application
-Before running this example, you need to build the application:
-
-```bash
-$ mvn package
-```
-This creates a JAR file with all dependencies embedded: `target/HelloModule-1.0-SNAPSHOT.jar`
+2. Download or clone the repository and navigate into the `native-hello-module` directory:
+    ```bush
+    git clone https://github.com/graalvm/graalvm-demos
```
Comment (olyagpl) [inline native-hello-module/README.md:29]: Split into 2 commands.

File: native-hello-module/README.md [hunk 4]
```diff
@@ -66,23 +50,34 @@ The following modularized Java Hello World example
...
-    [hellomodule:10847]      [total]:  19,216.19 ms,  4.85 GB
+3. Build this module into a native executable:
+    ```bush 
+    $JAVA_HOME/bin/native-image --module-path target/HelloModule-1.0-SNAPSHOT.jar --module HelloModule
+    ```
                                       
-It builds the modular Java app into a native executable that can be executed with:
-    
-    $ ./hellomodule 
+    It builds the modular Java application into a native executable called `hellomodule` in the project root directory that you can execute:
```
Comment (olyagpl) [inline native-hello-module/README.md:73]: Change to:

It builds the modular Java application into a native executable called `hellomodule` in the project root. Run it:

**Review by olyagpl — CHANGES_REQUESTED** [review]

File: native-hello-module/README.md (hunk 2 above)
Comment (LesiaChaban) [reply native-hello-module/README.md:15]: I will do it in  the next PR!

File: native-hello-module/README.md [hunk 5]
```diff
@@ -1,54 +1,35 @@
...
+# Build a Java Module into a Native Executable
 
-## Prerequisites
-* [GraalVM](http://graalvm.org)
-* [Native Image](https://www.graalvm.org/docs/reference-manual/native-image/)
-* Maven
+GraalVM Native Image supports the Java Platform Module System, introduced in Java 9, which means you can convert a modularized Java application into a native executable.
 
-## Preparation
+The `native-image` tool accepts the module-related arguments like `--module (-m)`, `--module-path (-p)`, `--add-opens`, `--add-exports` (same as for the java launcher). When such a module-related argument is used, the native-image tool itself is used as a module too.
```
Comment (olyagpl) [inline native-hello-module/README.md:5]: The `native-image` tool isn't wrapped in code tags here too:

... the native-image tool itself is used as a module too

File: native-hello-module/README.md [hunk 6]
```diff
@@ -1,54 +1,35 @@
...
+## Preparation
 
-3. Download or clone the repository and navigate into the `native-hello-module` directory:
-  ```bash
-  git clone https://github.com/graalvm/graalvm-demos
-  cd graalvm-demos/native-hello-module
-  ```
-## Build the Application
-Before running this example, you need to build the application:
+1. Download and install the latest GraalVM JDK with Native Image using [GraalVM JDK Dowloader](https://github.com/graalvm/graalvm-jdk-downloader):
```
Comment (olyagpl) [inline native-hello-module/README.md:17]: using using [GraalVM JDK Dowloader] > using **the** [GraalVM JDK Dowloader]

File: native-hello-module/README.md (hunk 5 above)
Comment (olyagpl) [inline native-hello-module/README.md:5]: The native-image tool isn't wrapped in code tags here:

... the native-image tool itself is used as a module too

File: native-hello-module/README.md (hunk 6 above)
Comment (olyagpl) [inline native-hello-module/README.md:17]: using using [GraalVM JDK Dowloader] > using **the** [GraalVM JDK **Downloader**]
(see the spelling mistake)

**Review by olyagpl — CHANGES_REQUESTED** [review]

**Review by olyagpl — APPROVED** [review]

**Pull request merged**